reporting/report.go

```diff
@@ -41,6 +41,11 @@ func (r *Report) ShowReport(format string, cfg *config.Config) error {
 
 func (r *Report) WriteFile(reportPath []string, cfg *config.Config) error {
 	for _, path := range reportPath {
+		slashPath := filepath.ToSlash(path)
```

Comment:
Can you please explain to me why do we need `ToSlash`?

Reply:
I was initially doing string manipulation before I found `filepath.Dir`, so wanted to make sure I'm being cross-platform. But now it's no longer needed, I'll remove it.